product-service-description/compliance-as-a-service/produkte/platform-linux.md

```diff
@@ -12,8 +12,33 @@ Other than that, no specific software needs to be installed (aside from the depe
 
 ## Supported packages
 
-- Both Entity Framework and SQLite queues can be launched on Linux, starting from version 1.3.3. 
-- Currently, SCU support is limited to the fiskaly and the Cryptovision packages, starting from version 1.3.3. Other SCUs (like Swissbit) will be supported soon.
+### Local storage support
+
+|                                                              | MX Linux   | Manjaro    | Linux Mint | Ubuntu     | Debian     |
+| ------------------------------------------------------------ | ---------- | ---------- | ---------- | ---------- | ---------- |
+| local mySQL Storage                                          | not tested | not tested | not tested | not tested | not tested |
```

Comment:
I'd shrink this list down to OSs that we actually tested and go with the following list:
- Ubuntu (tested: MySQL, SQLite)
- Debian (tested: MySQL, SQLite)

MX Linux and Majara are IMO anyway not used in this context, Linux Mint is just another Debian derivation (like Ubuntu, just a different UI)

Reply:
We could add a paragraph below or above that states that technically, it should work on all distributions that use Mono, but we haven't had the chance yet to do test this in detail.

Reply:
done